utils/src/parallel_restore.c

```diff
@@ -1832,13 +1832,55 @@ spr_err_t scoutfs_parallel_restore_write_buf(struct scoutfs_parallel_restore_wri
 	return count > 0 ? 0 : err;
 }
 
+/*
+ * Here we take in a dev's fd an read its quorum blocks to see if the dev has
+ * been mounted before
+ */
+static spr_err_t scoutfs_check_if_previous_mount(int fd)
+{
+	struct scoutfs_quorum_block *blk = NULL;
+	struct scoutfs_quorum_block_event *ev;
+	u64 blkno;
+	int i, j;
+	spr_err_t err;
+
+	for (i = 0; i <  SCOUTFS_QUORUM_MAX_SLOTS; i++) {
+		blkno = SCOUTFS_QUORUM_BLKNO + i;
+		err = read_block(fd, blkno, SCOUTFS_BLOCK_SM_SHIFT, (void **)&blk);
+		if (blk == NULL;)
```

Comment:
still has the semicolon here... please remove it.

or just write `if (!blk)`

Reply:
Hmm I was pretty certain that this got fixed in my changes but it doesn't seem so, I'll make sure its fixed in the next push